Keithley2612B_voltage_sweep_GUI.py

```diff
@@ -146,32 +174,33 @@ def save():
 frame_ic = LabelFrame(root, text = "INSTRUMENT CONTROL")
 frame_ic.grid(row = 0, column = 1, sticky = 'n')
 rm = kvs.get_resources()[0]
-address_list = kvs.get_resources()[1]
-
+address_list = kvs.get_resources()[1][0]
```

Comment:
You have used [0] here, are we sure that the VISA address of the sourcemeter will always be the first element of the address list? 
I wanted to put the address list array to the drop down menu from where the user can choose the address corresponding to the sourcemeter.